tests/test_make.py

```diff
@@ -676,6 +677,40 @@ def test_fields_properties(self, C):
             assert getattr(fields(C), attribute.name) is attribute
 
 
+class TestFieldsDict(object):
+    """
+    Tests for `fields_dict`.
+    """
+    def test_instance(self, C):
+        """
+        Raises `TypeError` on non-classes.
+        """
+        with pytest.raises(TypeError) as e:
+            fields_dict(C(1, 2))
+
+        assert "Passed object must be a class." == e.value.args[0]
+
+    def test_handler_non_attrs_class(self, C):
+        """
+        Raises `ValueError` if passed a non-``attrs`` instance.
+        """
+        with pytest.raises(NotAnAttrsClassError) as e:
+            fields_dict(object)
```

Comment:
empty line before assert block would be nice

Reply:
want me to clean these up in the tests for `fields` too? that's where I copy/pasted from 😉

Reply:
Sure, #boyScout. :)

Reply:
(done ✔️)